src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -33,9 +35,11 @@ public class PullRequestHandler : IHttpHandler
     public void ProcessRequest(HttpContext context)
     {
         _gitRepoPath = context.Request.PhysicalApplicationPath + "..\\..\\..\\..";
+        string branchString = context.Request.QueryString["branch"];
```

Comment:
Question only: I believe QueryString[] returns null if the given query parameter is not present.  Is that correct?  And if so, is HtmlEncode and other uses robust against nulls?